query/graphql/pom.xml

```diff
@@ -156,9 +152,6 @@
                             *
                         </Import-Package>
                         <Embed-Dependency>
-                            graphql-java,
```

Comment:
Why don't we need to embed graphql-java anymore?

Reply:
I saw it has a valid manifest when I was figuring out how to remove lombok so I removed the embed and started it with the feature instead.